Documentation/MSTest-Migration-EditorConfig.txt

```diff
@@ -0,0 +1,125 @@
+# Example .editorconfig for migrating from Philips MSTest analyzers to Microsoft's official MSTest analyzers
+# This configuration demonstrates the recommended migration settings
+
+# ==================================================
+# MICROSOFT MSTEST ANALYZERS (Recommended)
+# ==================================================
+# Install: <PackageReference Include="MSTest.Analyzers" Version="3.10.0" PrivateAssets="all" />
```

Comment:
Remove this Install line. People know how to do it, and the "3.10" will get out of date.